README.md

```diff
@@ -226,6 +226,9 @@ Changes to X or Y force recomputation of all results when a sample is rerun usin
 
 When releasing a new version, please add a Git tag of the form `vX.Y.Z`.
 
+- 4.8.0
+  - Fix counting of reads that have a tax ID but do not have an accession.
```

Comment:
would it be possible to add a bit more detail?
i.e. "...to ensure all reads mapped to taxa only by assembly are counted in r value."

Reply:
of course. just added.